conflux/adapter.js

```diff
@@ -0,0 +1,88 @@
+const Web3Utils = require('web3-utils')
```

Comment:
If possible, please make use of `ethers` utils, to be consistent with the rest of the repo